minecraft_servers/asvadianetwork/manifest.json

```diff
@@ -0,0 +1,30 @@
+{
+  "server_name": "asvadianetwork",
+  "nice_name": "Asvadia Network",
+  "direct_ip": "mc.asvadia.eu",
+  "server_wildcards": [
+    "%.asvadia.eu"
+  ],
+  "social": {
+    "web": "https://asvadia.eu",
```

Comment:
Not released, please remove

<img width="617" alt="Bildschirmfoto 2023-05-10 um 21 03 20" src="https://github.com/LabyMod/server-media/assets/45363287/97ce18d8-a8b0-4ba4-ab34-353c72d69e48">